lib/web3/formatters.js

```diff
@@ -112,6 +112,20 @@ var inputTransactionFormatter = function (options){
     return options;
 };
 
+/**
+ * Formats the input of the eth_getLogs command
+ *
+ * @method inputGetLogsFormatter
+ * @param {Object} getLogs options
+ * @returns {Object}
+ */
+var inputGetLogsFormatter = function (options) {
+    if (options.fromBlock)
+        options.fromBlock = inputBlockNumberFormatter(options.fromBlock)
+    if (options.toBlock)
+        options.toBlock = inputBlockNumberFormatter(options.toBlock)
```

Comment:
I think these two were missing the `;` according to lint, sorry @frozeman for that.